SCTMExecutor.err.noResponseFromSCTM=FEHLER: Keine Antwort von SCTM. Letzte Aktion fehlgeschlagen.
SCTMExecutor.err.notAUpstreamJob=FEHLER: Es besteht keine Abhängigkeit zum angegebenen Projekt ({0}). Überprüfe die Konfiguration!
SCTMExecutor.err.remoteConectionFailed=FEHLER: Das Aufbauen einer Serviceverbindung zu {0} ist fehlgeschlagen. Überprüfe die globalen Hudson Einstellungen!
SCTMExecutor.log.successfulLogin=INFO: An SilkCentral Test Manager anmelden.
SCTMExecutor.log.slaveNotConnected=FEHLER: Ergebnis kann nicht geschrieben werden weil kein Slave verbunden ist.
SCTMExecutor.msg.info.addBuildNumber=INFO: Füge Buildnummer  ''{0}'' zu SCTM Produkt hinzu.
SCTMExecutor.msg.info.buildnumberExists=INFO: Buildnummer ''{0}'' existiert bereits.

StdXMLResultWriter.testresult.failure=SCTM Fehler

SCTMResultWriter.err.createResultFolderExecDef=Das Verzeichnis für die Ergebnisse der Ausführungsdefinition ''{0}'' ({1}) kann nicht angelegt werden. 
SCTMResultWriter.err.err.createResultFolderTestDef=Das Verzeichnis für die Ergebnisse der Testdefinition ''{0}'' ({1}) kann nicht angelegt werden. 

ExecutionRunnable.log.resultReceived=INFO: Testergebnis für Ausführungsdefinition ''{0}'' ({1}) aufgezeichnet.
ExecutionRunnable.err.collectingResultsFailed=FEHLER: Ausführungsdefinition ''{0}'' ({1}) fehlgeschlagen. [{2}]
ExecutionRunnable.err.sessionLost=FEHLER: Session abgelaufen - Es ist nicht möglich eine neue Session zu öffen. Aktion wird abgebrochen. [{0}]
ExecutionRunnable.err.startExecDefFailed=FEHLER: Das Starten einer Ausführungsdefinition/Verzeichnisses ''{0}'' ({1}) ist fehlgeschlagen. SCTM meldet: {2}
ExecutionRunnable.msg.startExecDef=INFO: Start Ausführungs Definition ''{0}'' ({1}).
ExecutionRunnable.msg.waitForResult=INFO: Warte auf Ergebnis von Ausführungs Definition ''{0}'' ({1}).
ExecutionRunnable.warn.abortCollectingResults=WARNUNG: Das Sammeln der Ausführungsergebnisse für ''{0}'' ({1}) wurde abgebrochen.

TestConnectionValidator.msg.connectionFailed=Verbindung zu SilkCentral TestManager fehlgeschlagen. Überprüfen sie die Verbindungsdaten.
TestConnectionValidator.msg.connectionSuccessful=Verbindung erfolgreich hergestellt.
TestConnectionValidator.msg.wrongLoginData=Benutzer oder Passwort falsch.

SCTMExecutorDescriptor.plugin.title=SilkCentral Test Manager Ausführung
SCTMExecutorDescriptor.validate.msg.noValidURL=Keine gültige URL.
SCTMExecutorDescriptor.validate.msg.noServiceFound=Verbindung hergestellt; aber es konnte das benötigte Service nicht gefunden werden.